tests/src/Tables/TestCase.php

```diff
@@ -11,6 +12,7 @@ protected function getPackageProviders($app): array
         return [
             ...parent::getPackageProviders($app),
             TablesServiceProvider::class,
+            PowerJoinsServiceProvider::class,
```

Comment:
Can you move this to the base test case please? There is reasonable possibility we will need this elsewhere in the future.